components/Form/PhoneInput/PhoneInput.tsx

```diff
@@ -13,8 +13,8 @@ const PhoneInput = ({ rules, ...props }: Props): React.ReactElement => (
       // this should be removed when the BE
       // fix the inconsistencies in phone numbers
       pattern: {
-        value: /^[\d\s+]+$/,
-        message: 'Move text to phone type box',
+        value: /^[\d]+$/,
```

Comment:
This now doesn't match `+447890123456`, which is possibly a valid input.

We should double check if we need to support country code.

Separately, I'd expect a test change to be in this PR alongside a code change.